recipes/ninja/bld.bat

```diff
@@ -0,0 +1,5 @@
+%PYTHON% configure.py --bootstrap
+if errorlevel 1 exit 1
+
+COPY ninja.exe %LIBRARY_BIN%\ninja.exe
```

Comment:
We are usually OK with that, but open an issue at the `feedstock` as soon as it is created mentioning a possible build from source.